README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
@estute ah understood. We had in mind the following:
paver e2e_test (all tests)
paver e2e_test studio (tests in studio folder)
paver e2e_test whitelabel (tests in whitelabel folder)
paver e2e_test lms (tests in lms folder) 
So anyone running tests has all the options. But your idea is correct too, it's a rare chance that someone might need to run all the tests. I think we can do this too i.e --exclude-wl by default.
We can choose any of the two.